api/stream-legal-entity/openapi.yaml

```diff
@@ -956,6 +956,9 @@ components:
           $ref: '#/components/schemas/ExternalIdentifier'
         legalEntityType:
           $ref: '#/components/schemas/LegalEntityType'
+        createInDBS:
```

Comment:
I don't like the name of this flag, perhaps `readOnly` could be more elegant?
Please create a `description` in the specification itself for it

Reply:
added description. on name change we can discuss please.